src/index.js

```diff
@@ -172,62 +175,33 @@ State.prototype = {
 
     // Return the new state
     return state;
-  },
+  }
 
   // Find a list of child states matching the next character
-  match: function(ch) {
-    // DEBUG "Processing `" + ch + "`:"
```

Comment:
Were these used by a tool somewhere? Not that I could find, and they were cluttering the code a bit. I can re-add them (possibly with a gulp task) if they're still useful.